lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -428,6 +428,23 @@ bool UsdMaya_ReadJob::Read(std::vector<MDagPath>* addedDagPaths)
         topImportedPaths.insert(usdRootPrim.GetPath());
     }
 
+    SdfPathSet   sdfImportedPaths;
+    MSdfToDagMap sdfToDagMap;
+    for (const UsdPrim prim : stage->TraverseAll()) {
+        sdfImportedPaths.insert(prim.GetPath());
+    }
+
+    TF_FOR_ALL(pathsIter, sdfImportedPaths)
+    {
```

Comment:
I don't understand why you need two cycles here. You don't need to accumulate sdfImportedPaths first and then loop over it. You can what you need inside the first  loop.

Reply:
I forgot to delete it...Thanks for pointing out